app/app.go

```diff
@@ -497,6 +514,20 @@ func New(
 // Name returns the name of the App
 func (app *App) Name() string { return app.BaseApp.Name() }
 
+// Info implements the abci interface. It overrides baseapp's Info method, essentially becoming a decorator
+// in order to assign TimeoutInfo values in the response.
+func (app *App) Info(req *abci.RequestInfo) (*abci.ResponseInfo, error) {
```

Comment:
@julienrbrt can you confirm that any of the old diffs from here aren't needed anymore? https://github.com/celestiaorg/celestia-app/blob/main/app/app.go#L558